src/main.js

```diff
@@ -119,6 +119,14 @@ const router = new VueRouter({
       props: true,
       name: 'weeks_container'
     },
+    // ====================== model validation ======================
+    { path: '/modelvalidation/:uc_name',
+      component: function (resolve) {
+        require(['components/modelvalidation/mv-form.vue'], resolve)
+      },
+      props: true,
+      name: 'mv_form'
```

Comment:
The same thing here `name: 'ta_form'`

Reply:
Let's try these changes and let me know if it works the same.

PS: I don't know how I can test your pull request (do you have an option on the right in the pull request to allow changes from contributors?)

Reply:
Good you mentioned that ("...modify the notebook before putting it in the collab"). That is precisely one of the things we had in mind. Could you direct me to an example of such a currently available usecase?